Source/Krypton Components/Krypton.Toolkit/Palette Builtin/Base/PaletteBase.cs

```diff
@@ -1743,24 +1742,36 @@ public virtual float BaseFontSize
             }
         }
 
-        [AllowNull, DefaultValue(typeof(Font), DEFAULT_FONT_STYLE)]
+        /// <summary>Gets or sets the base palette font.</summary>
+        /// <value>The base palette font.</value>
+        [AllowNull, Description(@"Gets or sets the base palette font."), DefaultValue(null)]
         public virtual Font? BaseFont
```

Comment:
Remove null able return,. as it will never be null based on the setter and getter4.

Reply:
Simplified setter/getter